Source/UnitEx.cpp

```diff
@@ -960,6 +960,23 @@ void UnitEx::SetLaunchPadCargo(map_id moduleType)
 	(*unitArray)[unitID].launchPadCargo = moduleType;
 }
 
+map_id UnitEx::GetConVecCargoTurret()
+{
+	if (!isInited) {
+		return static_cast<map_id>(HFLNOTINITED);
+	}
+
+	if (!IsLive() || (GetType() != mapConVec)) {
+		return static_cast<map_id>(-1);

```

Comment:
The other functions in UnitEx seem to return a -1 cast as a map_id. I went with this since it seemed standard. I had considered mapNone as an alternative...

Reply:
I think this should either return map_id.mapNone or throw an exception. Dead and non-convec units aren't carrying a kit, so "none" would be a valid response. Method users will not be expecting a bad value to be returned.

Reply:
I agree, `mapNone` would make more sense here, and likely be closer to expectation of users.